components/Ink/src/MDCInkView.m

```diff
@@ -153,16 +153,6 @@ - (void)startTouchBeganAnimationAtPoint:(CGPoint)point
     inkLayer.inkColor = self.inkColor;
     inkLayer.maxRippleRadius = self.maxRippleRadius;
     inkLayer.animationDelegate = self;
-
-    switch (self.inkStyle) {
-      case MDCInkStyleBounded:
-        self.clipsToBounds = YES;
-        break;
-      case MDCInkStyleUnbounded:
-        self.clipsToBounds = NO;
-        break;
-    }
```

Comment:
Will removing this change existing functionality?
Do we need to set self.clipToBounds in our inkStyle setter?

Reply:
No, this will not affect existing functionality. Any clients who are using inkStyle will be using legacy ink and the default ink is legacy ink.